src/test/kotlin/JsonDumper.kt

```diff
@@ -0,0 +1,60 @@
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import java.time.LocalDateTime
+import no.nav.syfo.Apprec
+import no.nav.syfo.CS
+import no.nav.syfo.Helsepersonell
+import no.nav.syfo.Ident
+import no.nav.syfo.Organisation
+import no.nav.syfo.apprec.ApprecStatus
+
+fun main() {
```

Comment:
Var denne for å lage en fil med en apprec-json..? Trengs den fortsatt, eller kan vi kanskje skrive den om sånn at den er enten en apprec-generator (hvis vi trenger det for andre tester), eller at den tester et eller annet? :)